src/utils/colonyMotions.ts

```diff
@@ -0,0 +1,101 @@
+import { defineMessage } from 'react-intl';
+
+export enum NetworkMotionState {
+  Null = 0,
+  Staking = 1,
+  Submit = 2,
+  Reveal = 3,
+  Closed = 4,
+  Finalizable = 5,
+  Finalized = 6,
+  Failed = 7,
+}
+
+export enum MotionState {
+  Motion = 'Motion',
+  StakeRequired = 'StakeRequired',
+  Voting = 'Voting',
+  Reveal = 'Reveal',
+  Objection = 'Objection',
+  Failed = 'Failed',
+  Passed = 'Passed',
+  Invalid = 'Invalid',
+}
+
+const MSG = defineMessage({
+  motionTag: {
+    id: 'dashboard.ActionsPage.motionTag',
+    defaultMessage: 'Motion',
+  },
+  stakeRequiredTag: {
+    id: 'dashboard.ActionsPage.stakeRequiredTag',
+    defaultMessage: 'Stake required',
+  },
+  votingTag: {
+    id: 'dashboard.ActionsPage.votingTag',
+    defaultMessage: 'Voting',
+  },
+  revealTag: {
+    id: 'dashboard.ActionsPage.revealTag',
+    defaultMessage: 'Reveal',
+  },
+  objectionTag: {
+    id: 'dashboard.ActionsPage.objectionTag',
+    defaultMessage: 'Objection',
+  },
+  failedTag: {
+    id: 'dashboard.ActionsPage.failedTag',
+    defaultMessage: 'Failed',
+  },
+  passedTag: {
+    id: 'dashboard.ActionsPage.passedTag',
+    defaultMessage: 'Passed',
+  },
+  invalidTag: {
+    id: 'dashboard.ActionsPage.invalidTag',
+    defaultMessage: 'Invalid',
+  },
+});
+
+export const MOTION_TAG_MAP = {
+  [MotionState.Motion]: {
+    theme: 'primary',
+    colorSchema: 'fullColor',
+    name: MSG.motionTag,
+  },
+  [MotionState.StakeRequired]: {
+    theme: 'pink',
+    colorSchema: 'fullColor',
+    name: MSG.stakeRequiredTag,
+  },
+  [MotionState.Voting]: {
+    theme: 'golden',
+    colorSchema: 'fullColor',
+    name: MSG.votingTag,
+  },
+  [MotionState.Reveal]: {
+    theme: 'blue',
+    colorSchema: 'fullColor',
+    name: MSG.revealTag,
+  },
+  [MotionState.Objection]: {
+    theme: 'pink',
+    colorSchema: 'fullColor',
+    name: MSG.objectionTag,
+  },
+  [MotionState.Failed]: {
+    theme: 'pink',
+    colorSchema: 'plain',
+    name: MSG.failedTag,
+  },
+  [MotionState.Passed]: {
+    theme: 'primary',
+    colorSchema: 'plain',
+    name: MSG.passedTag,
+  },
+  [MotionState.Invalid]: {
+    theme: 'pink',
+    colorSchema: 'plain',
+    name: MSG.invalidTag,
+  },
+};
```

Comment:
Hmmm....

I'm good with having a tag map, but I don't think it should live inside `utils`

It should be somewhere next to the motion component, so inside `ActionPage`

Reply:
Didint see any good place for it in there 🤔 any ideas?

Reply:
`dashboard/ActionsPage/staticMaps` just so that all are in the same place.

And yes I realize that this isn't the ideal solution, since these are not really actions anymore, but motions.